platform-services-handler/index.js

```diff
@@ -61,9 +61,9 @@ module.exports.handler = (event, context, cb) => {
 		eventVersion: '1.0',
 		eventID: 'shardId-000000000000:49574219880753003597816065353678073460941546253285588994',
 		eventName: 'aws:kinesis:record',
-		invokeIdentityArn: 'arn:aws:iam::302890901340:role/lambda_basic_execution',
+		invokeIdentityArn: 'arn:aws:iam::xxx:role/lambda_basic_execution',
```

Comment:
use config variables to set this (as we do for our other services)

Reply:
This is just an example event for reference with random account id. Just getting rid of all the account numbers from the code.